api/web3server.go

```diff
@@ -617,6 +623,62 @@ func (svr *Web3Server) getTransactionByHash(in interface{}) (interface{}, error)
 	return svr.getTransactionCreateFromActionInfo(actionInfos)
 }
 
+func (svr *Web3Server) debugTransactionByHash(in interface{}) (interface{}, error) {
```

Comment:
An unit test is needed for this api, including a successful exec and a failed one at least